etc/apache2/includes/instance-oo.include.template

```diff
@@ -77,8 +77,8 @@
     ProxyPassReverse        /elasticsearch       http://localhost:${DEPLOYMENT_ES_HTTP_PORT}
 
     # Forward request to OnlyOffice when the url start with the version scheme (see in side the Docker container for it)
-    # ex: /v5.2.8-24/*
-    ProxyPassMatch    "^\/(v[\d]+\.[\d]+\.[\d]+[\.|-][\d]+\/.*)$"       http://localhost:${DEPLOYMENT_ONLYOFFICE_HTTP_PORT}/$1 acquire=1000 retry=30
+    # ex: /5.3.0-243/*
+    ProxyPassMatch    "^\/([\d]+\.[\d]+\.[\d]+[\.|-][\d]+\/.*)$"       http://localhost:${DEPLOYMENT_ONLYOFFICE_HTTP_PORT}/$1 acquire=1000 retry=30
```

Comment:
perhaps we should keep the v optional for instances not yet migration to the last oo version like cp instances